packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -50,19 +54,17 @@ class _FloatingActionButtonState extends State<FloatingActionButton> {
     return new Material(
       color: materialColor,
       type: MaterialType.circle,
-      elevation: _highlight ? 12 : 6,
-      child: new ClipOval(
-        child: new Container(
-          width: _kSize,
-          height: _kSize,
-          child: new InkWell(
-            onTap: config.onPressed,
-            onHighlightChanged: _handleHighlightChanged,
-            child: new Center(
-              child: new IconTheme(
-                data: new IconThemeData(color: iconThemeColor),
-                child: config.child
-              )
+      elevation: _highlight ? config.highlightElevation : config.elevation,
+      child: new Container(
```

Comment:
I probably would have put the container outside the material, but this works too.